pkg/blockatlas/tx.go

```diff
@@ -21,13 +21,15 @@ const (
 	TokenTypeBEP2  TokenType = "BEP2"
 	TokenTypeTRC10 TokenType = "TRC10"
 
-	TxTransfer            TransactionType = "transfer"
-	TxNativeTokenTransfer TransactionType = "native_token_transfer"
-	TxTokenTransfer       TransactionType = "token_transfer"
-	TxCollectibleTransfer TransactionType = "collectible_transfer"
-	TxTokenSwap           TransactionType = "token_swap"
-	TxContractCall        TransactionType = "contract_call"
-	TxAnyAction           TransactionType = "any_action"
+	TxTransfer               TransactionType = "transfer"
+	TxMultiCurrencyTransfer  TransactionType = "multi_currency_transfer"
+	TxNativeTokenTransfer    TransactionType = "native_token_transfer"
+	TxTokenTransfer          TransactionType = "token_transfer"
+	TxCollectibleTransfer    TransactionType = "collectible_transfer"
+	TxTokenSwap              TransactionType = "token_swap"
+	TxContractCall           TransactionType = "contract_call"
+	TxAnyAction              TransactionType = "any_action"
+	TxMultiCurrencyAnyAction TransactionType = "multi_currency_any_action"
```

Comment:
Why calling `multi_currency_any_action` and not `multi_currency_transfer` ?

Reply:
I added two type of tx, `multi_currency_any_action` and `multi_currency_transfer`

Reply:
Why two types? you just need one `multi_token_transfer `

Reply:
For the staking operations, fee and reward can also be multiple currencies.